### PR TITLE
Updated keystone policy for mitaka

### PR DESCRIPTION
**Pull request by srsakhamuri**
Updated keystone policy file for mitaka

### CODE REVIEW

**Comment by furlongm** [comment]
Curious if we should now have a matching policy file for horizon?

I remember it used to be needed.

**Comment by srsakhamuri** [comment]
I think it uses this policy file automatically
https://github.com/bloomberg/chef-bcpc/blob/91ad73333f89182f3fe36865dea6757492417f70/cookbooks/bcpc/recipes/horizon.rb#L162

File: cookbooks/bcpc/attributes/keystone.policy.rb [hunk 1]
```diff
@@ -4,187 +4,225 @@
...
-  "identity:get_domain_config" => "rule:admin_required",
-  "identity:update_domain_config" => "rule:admin_required",
-  "identity:delete_domain_config" => "rule:admin_required"
+    "admin_required" => "role:admin",
+    "cloud_admin" => "role:admin and (token.is_admin_project:True or domain_name:local)",
+    "service_role" => "role:service",
+    "service_or_admin" => "rule:admin_required or rule:service_role",
+    "owner" =>  "user_id:%(user_id)s or user_id:%(target.token.user_id)s",
+    "admin_or_owner" => "(rule:admin_required and domain_id:%(target.token.user.domain.id)s) or rule:owner",
+    "admin_and_matching_domain_id" => "rule:admin_required and project_domain_id:%(domain_id)s",
```
Comment (furlongm) [inline cookbooks/bcpc/attributes/keystone.policy.rb]: Should this be domain_id?

**Comment by srsakhamuri** [comment]
Will be merged to l-t-m-migration branch, closing this.

**Pull request closed without merging**